src/api/java/baritone/api/pathing/goals/GoalNear.java

```diff
@@ -51,6 +54,27 @@ public double heuristic(int x, int y, int z) {
         return GoalBlock.calculate(xDiff, yDiff, zDiff);
     }
 
+    @Override
+    public double heuristic() {//TODO less hacky solution
+        int range = (int)Math.ceil(Math.abs(Math.sqrt(rangeSq)));
```

Comment:
Why are you abs-ing a sqrt???

Reply:
true point
guess I didn't notice that the value I get is already guaranteed to be positive